crates/starknet-server/tests/get_transaction_by_hash.rs

```diff
@@ -16,14 +16,11 @@ mod get_transaction_by_hash_integration_tests {
     use starknet_rs_providers::{
         MaybeUnknownErrorCode, Provider, ProviderError, StarknetErrorWithMessage,
     };
-    use starknet_rs_signers::{LocalWallet, SigningKey};
     use starknet_types::contract_class::Cairo0Json;
     use starknet_types::felt::Felt;
     use starknet_types::traits::ToHexString;
 
-    use crate::common::constants::{
-        CASM_COMPILED_CLASS_HASH, PREDEPLOYED_ACCOUNT_ADDRESS, PREDEPLOYED_ACCOUNT_PRIVATE_KEY,
-    };
+    use crate::common::constants::CASM_COMPILED_CLASS_HASH;
```

Comment:
Its from old PR, but what this class hash represents

Reply:
```rust
// Example contract - Cairo 1
pub const CAIRO_1_CONTRACT_PATH: &str = "test_data/rpc/contract_cairo_v1/output.json";
pub const CASM_COMPILED_CLASS_HASH: &str =
    "0x63b33a5f2f46b1445d04c06d7832c48c48ad087ce0803b71f2b8d96353716ca";
```
I'm guessing it's the casm hash of the cairo contract whose path is `CAIRO_1_CONTRACT_PATH`